SECURITY.md

```diff
@@ -6,18 +6,13 @@ vulnerabilities, as well as suggestions around hardening the software and our pr
 
 ## Reporting a suspected vulnerability
 
-It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until
-after they have been analysed and a fix is available.
+It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until after they have been analysed and a fix is available.
```

Comment:
Should we mention that automated security scan results and public CVEs in dependencies are not considered a security issue? They are pretty common to get filed.

Reply:
@jonkoops I believe this is covered by "If you are reporting known CVEs related to third-party libraries used in Keycloak, please [create a new GitHub issue](https://github.com/keycloak/keycloak/issues/new/choose)."

Reply:
Sounds good